src/libsyntax/ast.rs

```diff
@@ -725,34 +783,78 @@ pub struct Expr {
 pub enum Expr_ {
     /// First expr is the place; second expr is the value.
     ExprBox(Option<P<Expr>>, P<Expr>),
+    /// An array (`[a, b, c, d]`)
     ExprVec(Vec<P<Expr>>),
+    /// A function call
+    /// The first field resolves to the function itself,
```

Comment:
you need a line break between these two unless you want them to get rendered all together